cookbook/case_studies/ml_training/nlp_processing/word2vec_and_lda.py

```diff
@@ -262,16 +256,15 @@ def word_similarities(model_ser: FlyteFile[MODELSER_NLP], word: str):
 # should be small. You can try altering either ``SENTENCE_A`` or ``SENTENCE_B`` variables to be dissimilar
 # to the other sentence, and check if the value computed is larger.
 @task(cache_version="1.0", cache=True, limits=Resources(mem="600Mi"))
-def word_movers_distance(model_ser: FlyteFile[MODELSER_NLP]):
+def word_movers_distance(model_ser: FlyteFile[MODELSER_NLP]) -> float:
     sentences = [SENTENCE_A, SENTENCE_B]
     results = []
     for i in sentences:
         result = [w for w in utils.tokenize(i) if w not in STOPWORDS]
         results.append(result)
     model = Word2Vec.load(model_ser.download())
-    distance = model.wv.wmdistance(*results)
     logger.info(f"Computing word movers distance for: {SENTENCE_A} and {SENTENCE_B} ")
-    logger.info(f"Word Movers Distance is {distance}")
```

Comment:
since im returning the inference result, i have got rid of the log as it is a bit redundant imo